lib/ansible/modules/windows/win_msg.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Jon Hawkesworth (@jhawkesworth) <figs@unity.demon.co.uk>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+DOCUMENTATION = '''
+---
+module: win_msg
+version_added: "2.3"
+short_description: Sends a message to logged in users on Windows hosts.
+description:
+    - Wraps the msg.exe command in order to send messages to Windows hosts.
+options:
+  to:
+    description:
+      - Who to send the message to. Can be a username, sessionname or sessionid.
+    required: false
+    default: all logged in users on the windows target.
```

Comment:
This doesn't seem right. Default states the default value, not some documentation string.